games-action/abuse/abuse-0.8.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="2D side-scrolling platform game"
+DESCRIPTION="Abuse is a dark 2D side-scrolling platform game developed by Crack dot Com in 1995. It features beautiful lighting, realistic animation and nasty alien-like creatures to destroy. It is now maintained by Sam Hocevar in an attempt to prevent it from vanishing from the Internet."
+HOMEPAGE="http://abuse.zoy.org"
+COPYRIGHT="
+	2005-2001 Sam Hocevar
+	2001 Anthony Kruize
+	1995 Crack dot Com
+	1995 Bobby Prince
+	1995 Crack dot Com
+	1996-2008 Justin Cassidy
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://abuse.zoy.org/raw-attachment/wiki/download/abuse-0.8.tar.gz"
+CHECKSUM_SHA256="0104db5fd2695c9518583783f7aaa7e5c0355e27c5a803840a05aef97f9d3488"
+SOURCE_DIR="abuse-0.8"
+PATCHES="abuse-0.8.patchset"
+ARCHITECTURES="x86_gcc2"
+
+PROVIDES="
+	abuse = $portVersion
+	cmd:abuse = $portVersion
+	cmd:abuse_tool = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	lib:libSDL
+	lib:libSDL_mixer_1.2
+	lib:libGL
+	lib:libGLU
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libSDL
+	devel:libSDL_mixer_1.2
+	devel:libGLU
+	"
+
```

Comment:
There shouldn't be an empty line here.

```diff
-
```

Per [Guidelines#Ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)